linq4j/src/main/java/org/apache/calcite/linq4j/EnumerableDefaults.java

```diff
@@ -3829,6 +3829,12 @@ private boolean advance() {
       TInner right = rightEnumerator.current();
       TKey rightKey = innerKeySelector.apply(right);
       for (;;) {
+        // mergeJoin assumes inputs sorted in ascending order with nulls last,
```

Comment:
Is it documented anywhere?

Reply:
I'll add a comment about it in the javadoc of EnumerableDefaults#mergeJoin and EnumerableDefaults#MergeJoinEnumerator

Reply:
Comments have been added.

Reply:
The nulls order is actually defined by the `RelFieldCollation`, did we set it anywhere ?

Reply:
It's set by EnumerableMergeJoinRule:
https://github.com/apache/calcite/blob/e427180b6a55445fe246b00c60259a95f96bdbf2/core/src/main/java/org/apache/calcite/adapter/enumerable/EnumerableMergeJoinRule.java#L72

Reply:
Reasonable, can you add this comments into the EnumerableDefaults code block.

Reply:
@danny0405 , comment has been added in the javadoc of EnumerableDefaults#mergeJoin and EnumerableDefaults#MergeJoinEnumerator